webapp/src/components/AssetPage/SmartBadge/SmartBadge.tsx

```diff
@@ -4,13 +4,22 @@ import { SmartIcon } from 'decentraland-ui'
 import IconBadge from '../IconBadge'
 import { Props } from './SmartBadge.types'
 import './SmartBadge.css'
+import { Section } from '../../../modules/vendor/decentraland'
+import { locations } from '../../../modules/routing/locations'
```

Comment:
We have some (annoying maybe :P) [import order standards](https://www.notion.so/decentraland/dApps-Standards-760ee6551c9f4c00aa71f7d9c50a7646#8d48e37938124a41ac664112cb1e27f6) we should follow! (super sorry lol)

```suggestion
import { Section } from '../../../modules/vendor/decentraland'
import { locations } from '../../../modules/routing/locations'
import IconBadge from '../IconBadge'
import { Props } from './SmartBadge.types'
import './SmartBadge.css'
```